Sources/WalletConnectEcho/Echo.swift

```diff
@@ -19,7 +20,10 @@ public class Echo {
 
     /// Echo instance config method
     /// - Parameter clientId: https://github.com/WalletConnect/walletconnect-docs/blob/main/docs/specs/clients/core/relay/relay-client-auth.md#overview
-    static public func configure(clientId: String) {
-        Echo.config = Echo.Config(clientId: clientId)
+    static public func configure(
+        clientId: String,
+        echoHost: String = "echo.walletconnect.com"
```

Comment:
may be constant in Echo SDK? to reuse it

Reply:
good point, I will fix it